packages/gdrcopy.spec

```diff
@@ -150,9 +150,19 @@ cp -a $RPM_BUILD_DIR/%{name}-%{version}/src/gdrdrv/Makefile $RPM_BUILD_ROOT%{usr
 cp -a $RPM_BUILD_DIR/%{name}-%{version}/src/gdrdrv/nv-p2p-dummy.c $RPM_BUILD_ROOT%{usr_src_dir}/gdrdrv-%{version}/
 cp -a $RPM_BUILD_DIR/%{name}-%{version}/dkms.conf $RPM_BUILD_ROOT%{usr_src_dir}/gdrdrv-%{version}
 
+%if 0%{!?suse_version:1}
 # Install gdrdrv service script
 install -d $RPM_BUILD_ROOT/etc/init.d
 install -m 0755 $RPM_BUILD_DIR/%{name}-%{version}/init.d/gdrcopy $RPM_BUILD_ROOT/etc/init.d
+%else
+mkdir -p $RPM_BUILD_ROOT/etc/modprobe.d
+cat <<"EOF" > $RPM_BUILD_ROOT/etc/modprobe.d/50-gdrdrv.conf
+#options gdrdrv dbg_enabled=1
+#options info_enabled=1
+install gdrdrv PATH=$PATH:/bin:/usr/bin; /sbin/modprobe --ignore-install gdrdrv $CMDLINE_OPTS && rm -f /dev/gdrdrv && mknod -m 666 /dev/gdrdrv c $(gawk '/gdrdrv/{printf"%%s",$1}' /proc/devices) 0 && chgrp video /dev/gdrdrv
+remove gdrdrv PATH=$PATH:/bin:/usr/bin; /sbin/modprobe -ir gdrdrv && rm -f /dev/gdrdrv
+EOF
+%endif
```

Comment:
do you need to set non-default permissions on 50-gdrdrv.conf?

Reply:
Not sure what you mean, permissions on /etc/modprobe.d/50-gdrdrv.conf should be 0644.

Reply:
For example, in the RH case, `install -m 0755 blabla`.

In the SUSE case, you use cat to create `50-gdrdrv.conf`, which will have default permissions. I was suggesting to explicitly set the permissions after creating the file.

Reply:
thanks for the change